src/CppUTestExt/IEEE754ExceptionsPlugin.cpp

```diff
@@ -34,6 +34,53 @@ extern "C" {
     #include <fenv.h>
 }
 
+#if CPPUTEST_FENV_LACKS_SIGNALLING
+
+/*
+ * Missing implementations for Apple, MinGW and Watcom C.
+ */
+
+#if defined(__APPLE__)
+#define CONTROL_WORD __control
+#elif defined(__MINGW32__)
+#define CONTROL_WORD __control_word
+#elif defined(__WATCOMC__)
+#define CONTROL_WORD control_word
+#endif
+
+int feenableexcept (int excepts)
+{
+    fenv_t fenv;
+    unsigned short old_excepts;
+    unsigned short new_excepts = (unsigned short)excepts & FE_ALL_EXCEPT;
```

Comment:
@basvodde maybe I am blinded by reading too much C code, but I am really at loss what should make this line "C++ style"?